tests/fuzz/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+set(libs
+    mbedtls
+)
+
```

Comment:
Please add a short `README` or `README.md` in `tests/fuzz` that explains:

* That the code here was contributed by you (you definitely deserve credit for them!).
* That these programs are used by oss-fuzz (with references to what oss-fuzz are: assume that the reader isn't familiar with it).
* A short explanation of how to run the programs.